clients/da-vinci-client/src/main/java/com/linkedin/davinci/blobtransfer/BlobTransferUtil.java

```diff
@@ -117,4 +144,24 @@ public static BlobTransferManager<Void> getP2PBlobTransferManagerForServerAndSta
       return null;
     }
   }
+
+  /**
+   * Get or create a global traffic shaping handler for P2P blob transfers.
+   * This single handler will be used across all channels for both server/client side to enforce global rate limits.
+   *
+   * @param blobTransferClientReadLimitBytesPerSec read limit in bytes per second
+   * @param blobTransferServiceWriteLimitBytesPerSec write limit in bytes per second
+   * @return the global traffic shaping handler
+   */
+  static GlobalChannelTrafficShapingHandler getGlobalTrafficShapingHandler(
+      long blobTransferClientReadLimitBytesPerSec,
+      long blobTransferServiceWriteLimitBytesPerSec) {
+    LOGGER.info(
+        "Global traffic shaping configured with read limit: {} bytes/sec, write limit: {} bytes/sec",
+        blobTransferClientReadLimitBytesPerSec,
+        blobTransferServiceWriteLimitBytesPerSec);
+
+    return BlobTransferGlobalTrafficShapingHandlerHolder
```

Comment:
Do we need this function here?
Maybe we can just expose a `static` method in `BlobTransferGlobalTrafficShapingHandlerHolder` to return a singleton instance instead of spreading the logic in multiple places.